code/datums/uplink_item.dm

```diff
@@ -991,6 +991,7 @@ var/list/uplink_items = list()
 	reference = "BRHS"
 	item = /obj/item/weapon/storage/box/syndie_kit/hardsuit
 	cost = 8
+	excludefrom = list(/datum/game_mode/nuclear)
```

Comment:
Maybe to prevent newer players screwing their team over by blowing TCs on something silly? They cost as much as their direct upgrade (Elites), and they already get enough of these suits to outfit the whole team anyways, without spending the TCs.